thunder/tests/test_inplace_functionalization.py

```diff
@@ -477,15 +477,15 @@ def f(xs, ys, z):
 )
 def test_inplace_to_tensors_with_grad(executor, device, _):
     @torch.no_grad
-    def add_y(x, y):
-        x.add_(y, alpha=0.1)
+    def add_grad(x, y):
+        return x.add_(x.grad)
 
     @torch.no_grad
-    def add_grad(x, y):
-        x.add_(x.grad, alpha=0.1)
+    def add_y(x, y):
+        return x.add_(y, alpha=0.1)
 
-    for f in (add_y, add_grad):
-        jitted_f = executor.make_callable(f)
+    for fn in (add_grad, add_y):
```

Comment:
The order of functions is changed, does it affect anything? If not let's revert the changes to this file to keep the diff smaller.

Reply:
No the order does not matter, except for readability.  When I see something in not alphabetical order, I assume that means that the order ~ does ~ matter.  But ok, smaller diff...